### DISCUSSION TITLE
display float with trailing zeros

### DISCUSSION THREAD

**Discussion by 8LWXpg** [typst/typst#2265] [Q&A] [answered]
how to show float with trailing zeros?

```typst
#let arr = (
  1.00, 2.01, 3.20
)
#table(..arr.map(x => str(x)))
```

<img width="67" alt="image" src="https://github.com/typst/typst/assets/105704427/7f846cab-d877-4b40-98af-4b22903995ec">

**Comment by xkevio** [comment, marked as answer]
You can use the [oxifmt](https://github.com/typst/packages/tree/main/packages/preview/oxifmt/0.2.0) package, or for this small occasion, this small function:
```typ
#let float_to_string(f, decimal_digits: 2) = {
  let decimal_part = str(calc.round(calc.fract(f), digits: decimal_digits)).slice(2)
  let integer_part = calc.trunc(f)

  decimal_part += "0" * (decimal_digits - decimal_part.len())

  return str(integer_part) + "." + decimal_part
}
```

**Reply by 8LWXpg** [reply]
didn't notice there's a package, thanks!
